*******> update.9

 Author: Daniel R. Roe

 Date: March 12, 2025

 Program: cpptraj

 Description: Fixes cpptraj TNG cmake build when zlib has been specified.

--------------------------------------------------------------------------------
--- AmberTools/src/cpptraj/src/tng/CMakeLists.txt	2024-03-29 15:43:43.000000000 -0400
+++ AmberTools/src/cpptraj/src/tng/CMakeLists.txt	2025-03-12 09:47:04.192767331 -0400
@@ -27,6 +27,10 @@
 add_library(tng_io STATIC ${TNG_IO_SOURCES})
 make_pic_if_needed(tng_io)
 target_include_directories(tng_io PUBLIC .)
+if(zlib_ENABLED)
+        include_directories(${ZLIB_INCLUDE_DIRS})
+endif()
+
 
 # checks based on tng_io's cmake script
 check_all_includes(C inttypes.h)
